app/controllers/application_controller.rb

```diff
@@ -1,2 +1,15 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    User.first
+  end
```

Comment:
- [ ] [REQUIRED] Hello @stevenmukama , great job so far, however, according to the project requirements Devise provides a method to handle the `current_user` method so the `current_user` method in `ApplicationController` should be removed.